corehq/util/bounced_email_manager.py

```diff
@@ -137,6 +138,7 @@ def _get_aws_info(self, message, uid):
                     destination=mail_info.get('destination', []),
                 ))
         else:
+            datadog_counter('bounced_email_manager.unknown_notification_type')
```

Comment:
How did you evade the prefix enforcer (metric name should start with `commcare.`)? Tests should have failed, but maybe there are no tests that trigger these error conditions?

https://github.com/dimagi/commcare-hq/blob/786db7e000394bcef2cad353d68f0c1521d4dc34/corehq/util/datadog/gauges.py#L120-L125

Reply:
updated